### PR TITLE
storage: Create storage images commits repositories endpoints.

### PR DESCRIPTION
**Pull request by ldjebran**
Create storage images commits repositories endpoints for  Image builder access

Fixes THEEDGE-2729

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/routes/storage.go [hunk 1]
```diff
@@ -35,6 +37,10 @@ func setContextUpdateTransaction(ctx context.Context, installer *models.UpdateTr
 	return context.WithValue(ctx, updateTransactionKey, installer)
 }
 
+func setContextStorageImage(ctx context.Context, image *models.Image) context.Context {
+	return context.WithValue(ctx, storageImageKey, image)
+}
+
 // MakeStorageRouter adds support for external storage
 func MakeStorageRouter(sub chi.Router) {
```
Comment (mgold1234) [inline pkg/routes/storage.go:45]: can you please add a test for that function?

Comment (ldjebran) [reply pkg/routes/storage.go:45]: That function is fully covered , it's already called before each test in https://github.com/RedHatInsights/edge-api/blob/main/pkg/routes/storage_test.go#L48

each test is also testing that the route is working properly.

Comment (mgold1234) [reply pkg/routes/storage.go:45]: great

**Review by de1987 — APPROVED** [review]

**Comment by ldjebran** [comment]
@mergify refresh

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**